autopool/tf.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+'''Autopool: Adaptive pooling operators for multiple instance learning'''
+
+from tensorflow.keras import backend as K
+from tensorflow.keras.layers import Layer, InputSpec
+from tensorflow.keras import initializers
+from tensorflow.keras import constraints
+from tensorflow.keras import regularizers
+
+
+class AutoPool1D(Layer):
+    '''Automatically tuned soft-max pooling.  (tensorflow.keras implementation)
```

Comment:
tensorflow.keras implementation?

Reply:
Yes, as opposed to the keras implementation, or a low-level tensorflow implementation (Does not exist)